core/app/globals.css

```diff
@@ -3,19 +3,6 @@
 @tailwind utilities;
 
 :root {
-  --primary: 96 100% 68%;
-  --accent: 96 100% 88%;
-  --background: 0 0% 100%;
-  --foreground: 0 0% 7%;
-  --success: 116 78% 65%;
-  --error: 0 100% 60%;
-  --warning: 40 100% 60%;
-  --info: 220 70% 45%;
-  --contrast-100: 0 0% 93%;
-  --contrast-200: 0 0% 82%;
-  --contrast-300: 0 0% 70%;
-  --contrast-400: 0 0% 54%;
-  --contrast-500: 0 0% 34%;
```

Comment:
Moved out of CSS to a Typescript file so we can reuse these values in theme component registration to provide props with default values.

Reply:
I'm wondering if there's a better way than deleting things for when we need to separate Makeswift related integration code from main. Ideally we don't have to delete anything from the base repo, just add Makeswift related code. Maybe that's wishful thinking.

Reply:
I think this is a good goal and we can accomplish it by overriding these values with Makeswift CSS variables further down. That being said, sometimes conflicts are good as we would want to know when this list changes to that we can do the equivalent change on `integrations/makeswift`.